.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Hi Carsten, good to see you back.  Why are the yml files part of this issue?  They aren't part of the system property cleanup extension.  Did you perhaps mean to submit them in a separate issue/commit and happened to have them in your branch?

Reply:
Yes, didn't want them to be part of this commit. Still need to find out how to remove them from patch in the github web gui like it was possible in gerrit.

Not sure that I'm going to contribute a lot. But at least had some spare time to look at how things are working after the migration to github.

Reply:
I believe one way would be to git format-patch the one commit you want, then do a git reset --hard to the patch before, then do a git am <patch you saved to the branch.  You should then be able to do a git push --force of the new version of the branch and it should show up here sans the other stuff.  Not fond of github myself.

Reply:
Seems to work fine in the github web gui for the pull request. Before I was looking for this in the github app but havn't found it there.

Reply:
Hi Carsten, it seems to work fine, but I don't know how to squash your commits and we shouldn't merge adding/deleting files.  Please do a git rebase -i locally and squash all the 5 commits into one commit so the xml changes aren't even seen or mentioned.  Also the PR is marked as Draft so I can't merge it until it is moved out of Draft.  Did I mention I hate github??  I would have solved this in gerrit in a flash.

Reply:
I think it is much easier than you think. It is just a question of your configuration.
https://egghead.io/lessons/github-automatically-squash-commits-on-github-when-merging-pull-requests